data/addressbook.json

```diff
@@ -15,7 +15,7 @@
     "time" : "20",
     "isFavourite" : true,
     "grains" : [ ],
-    "vegetables" : [ "ing                     w many spaces,  30.0g" ],
+    "vegetables" : [ "ing       w many spaces,  30.0g" ],
```

Comment:
Do we need to remove the whitespaces here? 
Or would it be removed in the next iteration?

Reply:
For the benefit of everyone who might read this in future, it's okay to leave it here! The next time you open the app and use it, these spaces will automagically disappear due to the power of coding 💪